src/types.ts

```diff
@@ -41,6 +42,7 @@ export interface RtcStream {
   mediaTypes: MediaType[];
   mediaStream: MediaStream;
   alias?: string;
+  participantId?: string;
```

Comment:
is the participantId always there in the Sdp response or optional? This denotes it's optional

Reply:
As it exists now it's optional. If we added into publish() (see comment above) I think we could remove the optional.

Reply:
With this merged and deployed to prod, this can be required if we want. Likewise with `alias`

Reply:
The setupPeerConnection() function can't supply a participant ID in all cases. If you look at my comment above in publish() we'd need to make a small webrtc_manager change to return it from requestToPublish. Until then I think it has to remain optional.